src/features/auth/components/LoginForm.tsx

```diff
@@ -62,9 +47,8 @@ function LoginForm() {
         }
       )
       .then((user) => {
-        setTimeout(() => {
-          setUser(user);
-        }, 3000);
+        localStorage.setItem("user", JSON.stringify(user));
```

Comment:
again, better create uselocal storage hook, which acts like ordinary state hook but uses useeffect to set item in local storage